src/main.tsx

```diff
@@ -6,6 +6,8 @@ import { theme } from './theme.ts';
 import { register } from './services/serviceWorker.ts';
 import { ThemeProvider } from '@mui/material';
 
+import './App.css';
```

Comment:
You should use mui v5's styling system not a raw CSS file, e.g. you could use the `sx` property or create a proper theme I think,see https://mui.com/material-ui/customization/how-to-customize/